### ISSUE TITLE
Reorganize alerts for TiKV, PD and TiDB

### ISSUE THREAD

**Issue by breeswish** [pingcap/tidb#18939]
## Feature Request

**Is your feature request related to a problem? Please describe:**

Some alert rules are outdated for v3.0 and v4.0. Some alert rules are missing. All alert rules need clear descriptions.

**Describe the feature you'd like:**

- [ ] Review, reorganize and polish alerts for TiKV
- [ ] Review, reorganize and polish alerts for PD
- [ ] Review, reorganize and polish alerts for TiDB

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

**Comment by zz-jason** [comment]
There are some alert rules that also may not needed, I think.

**Comment by breeswish** [comment]
@zz-jason Agree. Documentations may also be outdated. I will drive this refinement when we are implementing DataDog / Kibana integration in the near future.
